api/src/test/java/org/openmrs/obs/BinaryDataHandlerTest.java

```diff
@@ -63,7 +63,7 @@ public void shouldNotSupportOtherViews() {
     }
     
 	@Test
-	public void saveObs_shouldRetrieveCorrectMimetype() throws IOException {
+	public void saveObs_shouldRetrieveCorrectMimetypeAndTitle() throws IOException {
```

Comment:
This test (and all tests that I added except those in the ImageHandlerTest) passed prior to my change, I just wrote the tests to confirm/lock in the existing behavior.